app/models/api_user.rb

```diff
@@ -1,6 +1,22 @@
 class ApiUser < User
   default_scope { where(api_user: true).order(:name) }
+  validate :reason_for_2sv_exemption_blank
+  validate :require_2sv_is_false
 
   DEFAULT_TOKEN_LIFE = 2.years.to_i
   AUTOROTATABLE_TOKEN_LIFE = 60.days.to_i
+
+private
+
+  def require_2sv_is_false
+    if require_2sv
+      errors.add(:require_2sv, "can't be true for api user")
+    end
```

Comment:
I wonder if these would be better written on a single line, e.g. `errors.add(:require_2sv, "can't be true for api user") if require_2sv`?